src/__tests__/native-tests.ts

```diff
@@ -0,0 +1,149 @@
+import { runInContext } from '../index'
```

Comment:
I'm getting a weird bug where parse errors on native sometimes get `Line -1`. Can you confirm?

![maim](https://user-images.githubusercontent.com/829070/52548302-d550c580-2e07-11e9-95bb-e65d173df824.png)

Reply:
This is actually not caused by the Source parser.

The source parser only checks if `let b = 0;` is valid syntax, which it is.

After that, the code to 'put back' b in the scope gets prepended like 

```javascript
// previous globals;
let b = previous value of b;
// current line
let b =0;
```

So it gets evaled and throws the mentioned error. 

Any error caused by eval currently has an unknown location (since providing the actual location wouldn't make sense as it would be the location of the error of the transpiled code).

Reply:
I see. Can we print Line unknown instead? Doesnt need to be in this PR.